app/models/audit.py

```diff
@@ -0,0 +1,28 @@
+from sqlalchemy import Column, Integer, String, ForeignKey, Table
+from sqlalchemy.orm import relationship
+from sqlalchemy.sql.sqltypes import DateTime
+
+from app.config.database import Base
+
+audit_users = Table(
```

Comment:
mislim da su obje tablice i 'audit_users' i 'audit_documents' nepotrebne
samo dodaš relationship i postaviš parametar single_parent na true ako je jednostruka veza na nekoj strani (pogledaj u documents model kak je to rijeseno)

Reply:
Ispravio sam kako si rekla, samo nisam 100% siguran jel dobro pa bih te zamolio da provjeriš to novo.